chef/data_bags/crowbar/migrate/nova/032_add_zvm_attributes.rb

```diff
@@ -0,0 +1,16 @@
+def upgrade ta, td, a, d
+  a['zvm'] = ta['zvm']
+  d['element_states'] = td['element_states']
+  d['element_order'] = td['element_order']
+  d['element_run_list_order'] = td['element_run_list_order']
+  return a, d
+end
+
+def downgrade ta, td, a, d
+  a.delete('zvm')
+  d['element_states'].delete('nova-multi-compute-zvm')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.